integration-tests/pom.xml

```diff
@@ -99,6 +99,7 @@
         <module>cache</module>
         <module>qute</module>
         <module>bootstrap-config</module>
+        <module>optaplanner-jsonb</module>
```

Comment:
Move this under the optaplanner-jackson module a few lines up - keep em together.